.github/scripts/close-jira-issues.sh

```
@@ -0,0 +1,59 @@
+  --data '{
+  "resolution": {
+    "name": "Closed"
+  },
+  "transition": {
+    "id": "51"
+  }
+}'
+  CLOSED+=("$issue_key")
+done
+
+echo "Skipped issues: ${SKIPPED[@]}"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)